### PR TITLE
Switch to quay.io. Fix typos.

### PR DESCRIPTION
**Pull request by danielhoherd**
## Description

Switch from docker hub to Quay.io. Fix some typos.

## 🎟 Issue(s)

Relates to astronomer/issues#1355

## 🧪  Testing

These are CI and code comment changes only, so if CI passes the same as it did before then things are good.

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -169,7 +169,7 @@ workers:
     pollingInterval: 5
 
     # How many seconds KEDA will wait before scaling to zero.
-    # Note that HPA has a seperate cooldwon period for scale-downs
+    # Note that HPA has a separate cooldwon period for scale-downs
```
Comment (vishwas-astro) [inline values.yaml:172]: cooldown instead of cooldwon

File: README.md [hunk 2]
```diff
@@ -205,7 +205,7 @@ helm install --name my-release \
 KEDA stands for Kubernetes Event Driven Autoscaling. [KEDA](https://github.com/kedacore/keda) is a custom controller that allows users to create custom bindings
 to the Kubernetes [Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).
 We've built an experimental scaler that allows users to create scalers based on postgreSQL queries. For the moment this exists
-on a seperate branch, but will be merged upstream soon. To install our custom version of KEDA on your cluster, please run
+on a separate branch, but will be merged upstream soon. To install our custom version of KEDA on your cluster, please run
```
Comment (ashb) [inline README.md:208]: @dimberman could you open a new PR and update this part of the doc now that your KEDA changes have been merged upstream

File: values.yaml [hunk 3]
```diff
@@ -76,35 +76,35 @@ allowPodLaunching: true
...
 # Astronomer Airflow images
 images:
   airflow:
-    repository: astronomerinc/ap-airflow
+    repository: quay.io/astronomer/ap-airflow
     tag: ~
     pullPolicy: IfNotPresent
   flower:
-    repository: astronomerinc/ap-airflow
+    repository: quay.io/astronomer/ap-airflow
```
Comment (danielhoherd) [inline values.yaml:91]: @ashb Is this right? Flower uses ap-airflow container?

Comment (schnie) [reply values.yaml:91]: It's right. Flower is a subcommand on airflow.

**Review by schnie — APPROVED** [review]

**Pull request merged**